sonar-plugin-api/src/main/java/org/sonar/api/server/rule/RulesDefinitionXmlLoader.java

```diff
@@ -202,10 +273,28 @@ private void processRule(RulesDefinition.NewRepository repo, SMInputCursor ruleC
         severity = StringUtils.trim(cursor.collectDescendantText(false));
 
       } else if (StringUtils.equalsIgnoreCase("cardinality", nodeName)) {
-        cardinality = Cardinality.valueOf(StringUtils.trim(cursor.collectDescendantText(false)));
+        template = Cardinality.MULTIPLE == Cardinality.valueOf(StringUtils.trim(cursor.collectDescendantText(false)));
```

Comment:
All the if in this method are uglys, one day we should find a way to improve this...